### DISCUSSION TITLE
🔨iLogtail开发者双周会（4月12日 19:00-20:00）

### DISCUSSION THREAD

**Discussion by henryzhx8** [alibaba/ilogtail#770] [Announcements]
### 会议信息
时间：4月12日 周三 19:00 - 20:00
会议号：728 220 67481
电话呼入：+862281944293（中国大陆）+862759771624（中国大陆）
入会链接：https://meeting.dingtalk.com/j/K8I6KFUOQXv

### 议题
1. iLogtail K8s下的动态Go Profile采集 @EvanLjp 
2. 字符串替换处理插件的设计与使用 @pj1987111 
3. ”社区面对面“ @全体社区成员

### 会议纪要
1. 少旋同学分享了有关iLogtail K8s下的动态Go Profile采集内容，就采集原理进行了简单描述，并实际展示了采集效果。关于此议题，社区就iLogtail可观测与profiling相结合的议题进行了广泛的交流。
2. 锦琛同学分享了有关字符串替换处理插件的设计与使用话题，详细介绍了需求的来源、插件的设计背景和细节，同时提供了一些简单的配置示例。除此以外，锦琛同学还就`processor_split_key_value`插件的优化做了相关的介绍。
代码PR：https://github.com/alibaba/ilogtail/pull/757
https://github.com/alibaba/ilogtail/pull/766
https://github.com/alibaba/ilogtail/pull/758
3. 社区核心成员就社区广泛讨论和好奇的问题进行了解答（答案详见视频回放），包括：
- 使用hostpath挂载日志文件的业务容器重启，是否会导致容器文件重复采集呢？
- 容器文件的采集原理

### 视频回放
[iLogtail开发者双周会—第九期](https://www.bilibili.com/video/BV1va4y1N78Y/?spm_id_from=333.999.0.0)

**Comment by pj1987111** [comment]
1 字符串替换处理插件 需求来源：
https://github.com/alibaba/ilogtail/discussions/402
2 processor_split_key_value插件优化 需求来源：
https://github.com/alibaba/ilogtail/discussions/558
3 processor_split_key_value支持引用符 需求来源：
https://github.com/alibaba/ilogtail/discussions/555
